modules/standard/Spawn.chpl

```diff
@@ -276,10 +276,10 @@ module Spawn {
        Causes a fatal error if the subprocess does not have a
        stdin pipe open.
      */
-    proc stdin {
-      _halt_on_launch_error();
+    proc stdin throws {
+      try _throw_on_launch_error();
       if stdin_pipe == false {
-        halt("subprocess was not configured with a stdin pipe");
+        throw new Error("subprocess was not configured with a stdin pipe");
```

Comment:
Seems like we shouldn't be throwing a bare Error in a standard module. I know you're working on the Error subclasses in parallel. This one could be a SystemError with EINVAL. This comment applies to the other "not configure with" errors.

Reply:
I'll do this one after I merge the `SystemError` hierarchy, so I can use `SystemError.fromSyserr(EINVAL)`.

Reply:
I hope you make a SystemError.fromSyserr(EINVAL, "some particular error message") work